packages/pluggableWidgets/video-player-web/src/VideoPlayer.xml

```diff
@@ -9,12 +9,12 @@
         <propertyGroup caption="General">
             <propertyGroup caption="Data source">
                 <property key="urlExpression" type="expression">
```

Comment:
I don't see any consistency checks added in this PR (altho it's described in the story), is that automatically triggered by the fact that this property is implicitly required?

Reply:
Yes.